### PR TITLE
Dynamic Ingestion of a Website

### PR DESCRIPTION
**Pull request by sukantadash**
https://issues.redhat.com/browse/REDSAIA-107

### CODE REVIEW

File: data-ingestion/kfp/templates/execute-website-ingestion-kubeflow-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
```
Comment (jland-redhat) [inline data-ingestion/kfp/templates/execute-website-ingestion-kubeflow-pipeline.yaml]: Minor request, do you mind making a folder called `task` and putting this and the other task in there?

**Review by jland-redhat — CHANGES_REQUESTED** [review]
Once we make the index dynamic on the KFP side we will also need to pass it in through tekton

File: data-ingestion/kfp/templates/execute-website-ingestion-kubeflow-pipeline.yaml (hunk 1 above)
Comment (sukantadash) [reply data-ingestion/kfp/templates/execute-website-ingestion-kubeflow-pipeline.yaml]: This request is implemented.

**Review by jland-redhat — APPROVED** [review]

**Review by jland-redhat — APPROVED** [review]

**Comment by Jaland** [comment]
If you can fix the conflicts Sukanta I will get this guy merged in.

**Comment by sukantadash** [comment]
Need write access to this repo to merge the PR.

**Comment by Jaland** [comment]
Gave you member access to the org.  But the changes are on your personal fork. So you need to pull the upstream to fix the conflicts.

**Pull request merged**